### ISSUE TITLE
Release v0.8 properly, so that conda-forge can be updated

### ISSUE THREAD

**Issue by corneliusroemer** [brodie/cram#41]
It'd be great if you could release v0.8 properly ~via Github releases so the version can get bumped on conda-forge, among others.~ to PyPI so that it can be installed via PIP and conda-forge can also update the version. Maybe also make a Github release just for completeness.

Thanks!

**Comment by tsibley** [comment]
@corneliusroemer can correct me if I'm wrong, but I think there's two asks here (neither of which are actually making a GitHub Release™):

- Upload 0.8 to PyPI. This is [what conda-forge cares about](
https://github.com/conda-forge/cram-feedstock/blob/master/recipe/meta.yaml).
- Push a `0.8` tag matching what's in https://bitheap.org/cram/cram-0.8.tar.gz. This will help folks looking at tags on GitHub.

**Comment by corneliusroemer** [comment]
You're right @tsibley, I assumed it was pulling from Github releases, but it isn't. Recipe is set up to pull from PyPI 👼
